rero_ils/modules/local_entities/jsonschemas/local_entities/local_entity-v0.0.1.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Local entity",
+  "description": "JSON schema for a local entity",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "preferred_name",
+    "authorized_access_point",
+    "type"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate local entity records against.",
+      "type": "string",
+      "minLength": 9
+    },
+    "pid": {
+      "title": "Local entity PID",
+      "type": "string"
+    },
+    "preferred_name": {
+      "title": "Preferred name",
+      "type": "string",
+      "minLength": 1
+    },
+    "authorized_access_point": {
```

Comment:
This will be computed thus it will not be in the jsonschema.

Reply:
done